.github/workflows/release.yml

```diff
@@ -44,21 +44,10 @@ jobs:
       - name: Build 'kolena' Python package
         run: poetry build --format=sdist
 
-      - name: Build 'kolena' documentation and push to S3
+      - name: Build 'kolena' documentation and push to GitHub Pages
         run: |
-          ./render.sh
-          VERSION=$(poetry run python3 -c "import kolena; print(kolena.__version__)")
-
-          BUCKET_PATH="s3://$ASSET_BUCKET/docs/$VERSION"
-          echo "pushing documentation to '$BUCKET_PATH'..."
-          aws s3 rm --recursive "$BUCKET_PATH"
-          aws s3 sync ./build "$BUCKET_PATH"
-
-          BUCKET_PATH="s3://$ASSET_BUCKET/docs/latest"
-          echo "pushing documentation to '$BUCKET_PATH'..."
-          aws s3 rm --recursive "$BUCKET_PATH"
-          aws s3 sync ./build "$BUCKET_PATH"
-        working-directory: ./docs
+          poetry run mkdocs build
+          poetry run mkdocs gh-deploy
```

Comment:
Non-blocking note: Need to somehow protect this branch. I just tried to run this command and the branch got created.

Reply:
👍 will add protections. Want to run this path once in CI to see how the commits look, then restrict the branch to only accept that sort of commit.